### PR TITLE
{bio}[foss/2018b] cutadapt v2.3 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@pescobar please add:
```
use_pip = True
download_dep_fail = True
```

File: easybuild/easyconfigs/c/cutadapt/cutadapt-2.3-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['98b864ac9d81525263897bdda9ec5bfd4ff6b5d2ef48f915219def620a6e92ac']
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
```
Comment (smoors) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-2.3-foss-2018b-Python-3.6.6.eb]: please add:
```
use_pip = True
download_dep_fail = True
```

Comment (smoors) [reply easybuild/easyconfigs/c/cutadapt/cutadapt-2.3-foss-2018b-Python-3.6.6.eb]: extensions xopen and dnaio are not needed anymore?

**Comment by boegel** [comment]
@pescobar The existing easyconfig we have for `cutadapt` 2.1 uses `PythonBundle` and also installs the `xopen` and `dnaio` Python packages as requirements for `cutadapt`.

Are you sure those are no longer needed?

**Comment by boegel** [comment]
Closing this, since i) this is used a [deprecated toolchain](https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains), ii) there was no reply for over 3 years

**Pull request closed without merging**